src/jsep.js

```diff
@@ -1,258 +1,427 @@
 //     JavaScript Expression Parser (JSEP) <%= version %>
 //     JSEP may be freely distributed under the MIT License
 //     https://ericsmekens.github.io/jsep/
+export class Jsep {
+	/**
+	 * @returns {string}
+	 */
+	static get version() {
+		// To be filled in by the template
+		return '<%= version %>';
+	}
+
+	/**
+	 * @returns {string}
+	 */
+	static toString() {
+		return 'JavaScript Expression Parser (JSEP) v' + Jsep.version;
+	};
+
+	// ==================== CONFIG ================================
+	/**
+	 * @method addUnaryOp
+	 * @param {string} op_name The name of the unary op to add
+	 * @returns {Jsep}
+	 */
+	static addUnaryOp(op_name) {
+		Jsep.max_unop_len = Math.max(op_name.length, Jsep.max_unop_len);
+		Jsep.unary_ops[op_name] = 1;
+		return Jsep;
+	}
+
+	/**
+	 * @method jsep.addBinaryOp
+	 * @param {string} op_name The name of the binary op to add
+	 * @param {number} precedence The precedence of the binary op (can be a float)
+	 * @returns {Jsep}
+	 */
+	static addBinaryOp(op_name, precedence) {
+		Jsep.max_binop_len = Math.max(op_name.length, Jsep.max_binop_len);
+		Jsep.binary_ops[op_name] = precedence;
+		return Jsep;
+	}
+
+	/**
+	 * @method addIdentifierChar
+	 * @param {string} char The additional character to treat as a valid part of an identifier
+	 * @returns {Jsep}
+	 */
+	static addIdentifierChar(char) {
+		Jsep.additional_identifier_chars.add(char);
```

Comment:
Does this function only exist for backwards compat? Because if `additional_identifier_chars` is exposed, people can just add characters to it directly.

Reply:
Agreed.